awswrangler/timestream.py

```diff
@@ -668,12 +676,12 @@ def list_tables(database: Optional[str] = None, boto3_session: Optional[boto3.Se
     ... ["table1"]
 
     """
-    client: boto3.client = _utils.client(service_name="timestream-write", session=boto3_session)
+    client = _utils.client(service_name="timestream-write", session=boto3_session)
     args = {} if database is None else {"DatabaseName": database}
-    response: Dict[str, Any] = client.list_tables(**args)
+    response = client.list_tables(**args)  # type: ignore[arg-type]
     tables: List[str] = [tbl["TableName"] for tbl in response["Tables"]]
     while "nextToken" in response:
-        response = client.list_tables(**args, nextToken=response["nextToken"])
```

Comment:
This was an actual bug fix. The error was surfaced by MyPy thanks to these stubs.